Python/flowgraph.c

```diff
@@ -953,6 +953,15 @@ label_exception_targets(basicblock *entryblock) {
            PyMem_Free(except_stack);
         }
     }
+
+    for (basicblock *b = entryblock; b != NULL; b = b->b_next) {
+        for (int i = 0; i < b->b_iused; i++) {
+            cfg_instr *instr = &b->b_instr[i];
+            if (instr->i_opcode == POP_BLOCK) {
+                INSTR_SET_OP0(instr, NOP);
```

Comment:
Could we do this in the main loop, at line 905?

Reply:
Probably. I'll try that.

Reply:
Yup, it works.